plugins/sudo.js

```
@@ -1,48 +1,87 @@
-		return message.send(formattedSudoList, { mentions: mentions });
-	},
+  {
+    pattern: 'addsudo',
+    public: false,
+    desc: 'Adds Set of Users as Sudo Users',
+    type: 'sudo',
+  },
+  async (message, match) => {
+    if (!match && !message.mention)
+      return message.send('_Usage: 12345567,22445,356 or @user1,@user2,@user3_');
+    match = match.replace(/@/g, '').split(/,\s*|\s+/g);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)